Documentation/Helm-Charts/operator-chart.md

```diff
@@ -147,6 +147,7 @@ The following table lists the configurable parameters of the rook-operator chart
 | `logLevel` | Global log level for the operator. Options: `ERROR`, `WARNING`, `INFO`, `DEBUG` | `"INFO"` |
 | `monitoring.enabled` | Enable monitoring. Requires Prometheus to be pre-installed. Enabling will also create RBAC rules to allow Operator to create ServiceMonitors | `false` |
 | `nodeSelector` | Kubernetes [`nodeSelector`](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector) to add to the Deployment. | `{}` |
+| `obcProvisionerNamePrefix` | Uses prefix instead of operator namespace to provisioner | `namespace name where rook-ceph operator is deployed` |
```

Comment:
```suggestion
| `obcProvisionerNamePrefix` | Uses prefix instead of operator namespace name in obc provisioner name  | `namespace name where rook-ceph operator is deployed` |
```